### DISCUSSION TITLE
Persistent vehicles

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1369] [Q&A] [answered]
Does SHVDN make DLC vehicles not despawn? If yes, is it in full [stable] release too?

**Comment by kagikn** [comment, marked as answer]
What full release mean, but SHVDN prevents DLC vehicles from despawning (by patching a piece of a ysc script of shop controls). You won't notice the difference as long as no DLC vehicles are spawned, unlike how RPH increases the max magnitude limits of linear and angular velocity for physical entities.

**Reply by Michael21107** [reply]
by full release, i mean a release in this repo

**Reply by kagikn** [reply]
ah, that's what I call stable releases, yes SHVDN should make DLC vehicles not despawn in these versions. Unless Rockstar alter their shop script and can't find the place to patch with the memory patterns/signatures we're using.

**Reply by Michael21107** [reply]
gotcha, may i ask from which stable version is it like that?

**Reply by kagikn** [reply]
The countermeasure (with future proof) is included since v3.1.0, but only v3.6.0 can do the job in b2802 or later game builds of all stable releases as of writing. Rockstar added opcodes for their ysc scripts and SHVDN couldn't find the address to patch with any of the existing memory patterns.

**Reply by Michael21107** [reply]
yep, thank you
